### ISSUE TITLE
Layouts not being rendered

### ISSUE THREAD

**Issue by stevenleeg** [samvincent/jekyll-haml#14]
Hey everyone,

I'm having an issue with my haml layouts not being rendered. My setup is a `_layouts/default.haml`:

```haml
!!!
%html
  %head
    %title title stuff
    [...]

  %body
    = "{{ content }}"
```

and an `index.haml`:

```haml
---
layout: default
---

%i Hello world!
```

When I run `jekyll serve --watch` and head over to localhost:4000 I see that the `default.haml` is just being outputted haml (it's not being compiled) `{{ content }}` is outputting the compiled haml as html (as it should).

Any ideas as to what would be causing this? I'm not sure if I overlooked something or not, so hopefully this is a silly mistake on my part...

Thanks in advance, and keep up the great work on this project!
– Steve

**Comment by davidpdrsn** [comment]
I'm having the same issue

**Comment by lewispb** [comment]
Same here

**Comment by keikun17** [comment]
@stevenleeg @davidpdrsn @lewispb : this PR fixes it https://github.com/yaegashi/jekyll-haml/commit/ce43db3506be17165f59df0ae969d1336768e6f8

**Comment by samvincent** [comment]
Version 0.1.2 published to rubygems.org

**Comment by dgmstuart** [comment]
@samvincent layouts aren't getting rendered for me - perhaps 0.1.3 broke it?

**Comment by zeroedin** [comment]
+1 for @dgmstuart , 0.1.3 appears to have broken layouts rendering with haml.

**Comment by samvincent** [comment]
Let me know if a pull request added to master corrects the issue. I don't have a project using this at the moment.

```
# Gemfile
gem 'jekyll-haml', :git => 'https://github.com/samvincent/jekyll-haml.git'
```

**Comment by sspreitzer** [comment]
Reopened duplicate #17 
Fix with PR #18

**Comment by sspreitzer** [comment]
PR merged, issue fixed.
